modules/theora/editor/rgb2yuv.h

```diff
@@ -0,0 +1,49 @@
+#ifndef RGB2YUV_H
+#define RGB2YUV_H
```

Comment:
This needs the Godot copyright header and switching to `#pragma once`:
https://github.com/godotengine/godot/actions/runs/15560984396/job/43812941752?pr=107188

You can do it easily with `pre-commit`:

```
pip install pre-commit
pre-commit install
pre-commit run --all
```